bindings/python/multibody/expose-liegroups.cpp

```diff
@@ -0,0 +1,62 @@
+//
+// Copyright (c) 2015-2020 CNRS INRIA
+//
+
+#include "pinocchio/bindings/python/fwd.hpp"
+#include "pinocchio/bindings/python/multibody/liegroups.hpp"
+
+#include <eigenpy/memory.hpp>
+
+namespace pinocchio
+{
+namespace python
+{
+
+namespace bp = boost::python;
+
+template<typename LgType>
+CartesianProductOperationVariantTpl<double,0,LieGroupCollectionDefaultTpl>
+makeLieGroup()
+{
+  return CartesianProductOperationVariantTpl<double,0,LieGroupCollectionDefaultTpl> (LgType());
+}
+
+CartesianProductOperationVariantTpl<double,0,LieGroupCollectionDefaultTpl>
+makeRn(int n)
+{
+  return CartesianProductOperationVariantTpl<double,0,LieGroupCollectionDefaultTpl> (
+      VectorSpaceOperationTpl<Eigen::Dynamic,double,0>(n));
+}
+
+
+/* --- Expose --------------------------------------------------------- */
+void exposeLieGroups()
+{
+  LieGroupPythonVisitor<
+    CartesianProductOperationVariantTpl<double,0,LieGroupCollectionDefaultTpl>
+    >::expose("LieGroup");
+
+  bp::scope current;
+  std::string submoduleName(bp::extract<const char*>(current.attr("__name__")));
+  submoduleName.append(".liegroups");
+
+  // Create the submodule, and attach it to the current scope.
```

Comment:
In the utils folder, there are some tools to do that automatically.

Reply:
Can you be more precise ?

Reply:
See https://github.com/stack-of-tasks/pinocchio/blob/a179f5619a24000d71d748243b5068605cce4453/bindings/python/utils/namespace.hpp#L23-L33